Sources/FoundationEssentials/Decimal/Decimal+Compatibility.swift

```diff
@@ -80,23 +80,13 @@ extension Decimal : _ObjectiveCBridgeable {
 // MARK: - Bridging code to C functions
 // We have one implementation function for each, and an entry point for both Darwin (cdecl, exported from the framework), and swift-corelibs-foundation (SPI here and available via that package as API)
 
-#if FOUNDATION_FRAMEWORK
 @available(macOS 10.10, iOS 8.0, watchOS 2.0, tvOS 9.0, *)
 public func pow(_ x: Decimal, _ y: Int) -> Decimal {
     let result = try? x._power(
         exponent: y, roundingMode: .plain
     )
     return result ?? .nan
 }
-#else
```

Comment:
Is `pow` available as API on `Decimal` itself instead of just via the free function?

Reply:
My read is that this has always been a free function, both on Darwin and non-Darwin. This PR just unifies the code and makes it available for FoundationEssentials too.

Reply:
Ditto